console.js

```diff
@@ -0,0 +1,32 @@
+import { format } from './util.js'
+
+const console = typeof window !== 'undefined' ? window.console ?? {} : {}
```

Comment:
```suggestion
const console = window?.console ?? {}
```

Reply:
that results in:

```js
file:///Users/werle/repos/socketsupply/socket-api/console.js:3
const console = window?.console ?? {} : {}
                                      ^

SyntaxError: Unexpected token ':'
    at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:115:18)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:289:14)
```

when importing it in node, which I do in `bin/repl.js`

Reply:
`: {}` is unnecessary there